web_app_drawer/views/web.xml

```diff
@@ -214,13 +209,16 @@
             
             <div class="panel-default app-drawer-app-panel" id="appDrawerAppMenu">
                 <div class="panel-heading" id="appDrawerAppPanelHead">
-                    <h3 class="panel-title">
+                    <h4 class="app-drawer-panel-title">
+                        <a href="#" class="app-drawer-icon-close">
+                            <i class="fa fa-chevron-left drawer-toggle" />
```

Comment:
Oops I think you've said this like 3-4 times now too. My bad - so many comments and tasks, so little brain 😛